Add venv to install instructions page (#3960)
fishtown-analytics/docs.getdbt.com: website/docs/docs/core/pip-install.md

You need to use `pip` to install dbt Core on Windows or Linux operating systems. You can use `pip` or [Homebrew](/docs/core/homebrew-install) for installing dbt Core on a MacOS. 

You can install dbt Core and plugins using `pip` because they are Python modules distributed on [PyPI](https://pypi.org/project/dbt/).

<FAQ path="Core/install-pip-os-prereqs" />
<FAQ path="Core/install-python-compatibility" />

### Using virtual environments

> We recommend using virtual environments to namespace pip modules. To create a new venv:

Reviewer: ```suggestion
We recommend using virtual environments (venv) to namespace pip modules. 

1. Create a new venv:
```

```shell
python3 -m venv dbt-env				# create the environment
```

> If you install dbt in a virtual environment, you need to reactivate that same virtual environment each time you create a shell window or session.

Reviewer: ```suggestion
2. Reactivate the dbt virtual environment with every new session. When you install dbt in a virtual environment, you need to reactivate that same virtual environment each time you open a new shell window or session.
```

Author: Changed this up a bit to be shorter.

```shell
source dbt-env/bin/activate			# activate the environment for Mac and Linux OR
dbt-env\Scripts\activate			# activate the environment for Windows
```

> :::tip

Reviewer: Let's use a heading so we can link to it above. And so others can link directly to it if necessary..
```suggestion
#### Create an alias
```

> You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever rc file your shell draws from. 

Reviewer: I suggest calling it a "config" which feels more universally understood. I know rc is technically more than a config, but hopefully config works here.
```suggestion
You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from. 
```

For example, add the following to your rc file, replacing <PATH_TO_VIRTUAL_ENV_CONFIG> with the path to your virtual environment configuration.

```shell
alias env_dbt='source <PATH_TO_VIRTUAL_ENV_CONFIG>/bin/activate'
```

> :::

Reviewer: Need to remove if we're using a heading.
```suggestion
```

You need to use `pip` to install dbt Core on Windows or Linux operating systems. You can use `pip` or [Homebrew](/docs/core/homebrew-install) for installing dbt Core on a MacOS. 

You can install dbt Core and plugins using `pip` because they are Python modules distributed on [PyPI](https://pypi.org/project/dbt/).

<FAQ path="Core/install-pip-os-prereqs" />
<FAQ path="Core/install-python-compatibility" />

### Using virtual environments
We recommend using virtual environments (venv) to namespace pip modules. 

1. Create a new venv:

```shell
python3 -m venv dbt-env				# create the environment
```

2. Activate that same virtual environment each time you create a shell window or session:

```shell
source dbt-env/bin/activate			# activate the environment for Mac and Linux OR
dbt-env\Scripts\activate			# activate the environment for Windows
```

#### Create an alias

> You can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from. 

Reviewer: ```suggestion
To activate your dbt environment with every new shell window or session, you can create an alias for the source command in your $HOME/.bashrc, $HOME/.zshrc, or whichever config file your shell draws from. 
```
